src/rust/zh.dmzj/res/source.json

```
@@ -3,7 +3,7 @@
 		"id": "zh.dmzj",
 		"lang": "zh",
 		"name": "动漫之家",
-		"version": 1,
+		"version": 2,
 		"url": "https://m.dmzj.com"
 	}
 }
```

should have a final newline for all files
```suggestion
}

```